source/jsonnet/england-wales/individual/blocks/qualifications/a_level.jsonnet

```diff
@@ -9,27 +9,39 @@ local proxyTitle = {
   ],
 };
 
-local englandGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside England and Wales';
-local walesGuidanceTitle = 'Include equivalent qualifications achieved anywhere outside Wales and England';
+local englandQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside England and Wales';
+local walesQuestionDescription = 'This could be equivalent qualifications achieved anywhere outside Wales and England';
+
+local englandGuidanceList = [
+  'These are advanced-level, subject-based qualifications that are often needed to get a place at university. Students in England and Wales usually complete AS levels by the age of 17 years and A levels by the age of 18 years.',
+  'If they have achieved similar qualifications outside of England and Wales, choose the options they think are the closest match. An International Baccalaureate diploma is equivalent to three A levels.',
+];
+local walesGuidanceList = [
+  'These are advanced-level, subject-based qualifications that are often needed to get a place at university. Students in Wales and England usually complete AS levels by the age of 17 years and A levels by the age of 18 years.',
+  'If you have achieved similar qualifications outside of Wales and England, choose the options you think are the closest match. An International Baccalaureate diploma is equivalent to three A levels.',
+];
```

Comment:
These is only 2 variations but it should be four. At the moment, its eng/wls but it should be, 
1. Eng proxy
2. Eng non proxy
3. Wls proxy
4. Wls non proxy

Reply:
This applies to more than just a levels.